Assets/Scripts/Core/GameManager.cs

```diff
@@ -143,10 +153,49 @@ public float GetFindingFloatDistance()
         return configurator.findingFloatDist;
     }
 
-    public float GetFindingsCount()
+    public float GetExtraBombFindingsCount()
     {
-        // TODO This method's result should be dependent on current level number value!
-        return configurator.initialFindingsCount;
+        if(extraBombFindingsCount < 0)
+        {
+            extraBombFindingsCount = configurator.initialExtraBombFindings;
+        }
+        return extraBombFindingsCount;
+    }
+
+    public float GetExtraLiveFindingsCount()
+    {
```

Comment:
Noi o to właśnie chodzi. Modyfikacja powinna zależeć od numeru aktualnego poziomu i od wartości dla poziomu początkowego.